app/Controllers/UsuarioController.cs

```diff
@@ -51,5 +53,39 @@ public IActionResult CadastrarUsuarioTerceiro([FromBody] UsuarioDTO usuarioDTO)
 
             return Ok();
         }
+
+        [HttpPut("recuperarSenha")]
+        public IActionResult RecuperarSenha([FromBody] UsuarioDTO usuarioDto)
+        {
+            try
+            {
+                usuarioService.RecuperarSenha(usuarioDto);
+                return Ok();
+            }
+            catch(KeyNotFoundException)
+            {
+                return NotFound();
+            }
+        }
+
+        //  [HttpPut("Validar token de recuperação")]
```

Comment:
Aqui vai uma dica: segundo Martin Fowler, não há nada pior do que deixar um código comentado. Se a gente deixar esse código comentado, a próxima pessoa que pegar esse código vai achar que teve algum motivo para deixar esse código comentado e não vai remover, assim vai acontecer com a próxima e a próxima... No final das contas, isso vira um código morto que poderia ter sido removido.

Trago essa reflexão pra ti: por qual motivo esse código esta comentado? Se tem um motivo, pq você não deixou explicito em um comentário? Se não tem, pq você não remove logo?

Fica aqui uma sugestão de arquivo que trás um resumo sobre essa parte de comentários no livro clean code -> https://betterprogramming.pub/15-rules-for-writing-clean-comments-18390394941f

Reply:
Removi, realmente não há necessidade de manter este comentário.